app/controllers/admin/dashboards_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::DashboardsController < Admin::ApplicationController
+  def index
```

Comment:
__[rails_best_practices]__
* move model logic into model (Slide use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)

<!-- Comment by [SideCI](https://sideci.com) -->